amundsen_application/static/js/components/NavBar/index.tsx

```diff
@@ -51,6 +51,10 @@ class NavBar extends React.Component<NavBarProps, NavBarState> {
         <div className="row">
           <div className="nav-bar">
             <div className="nav-bar-left">
+              {
+                AppConfig.logoPath &&
+                <img src={AppConfig.logoPath} />
```

Comment:
Might be nice to give this a class like "logo-icon" and give it some max width / height.

Reply:
No problem. Will add something that makes sense for now and we can tweak it later as our NavBar evolves.
```
.logo-icon {
  max-height: 48px;
  max-width: 144px;
}
```